clients/cobol-lsp-vscode-extension/syntaxes/COBOL.tmLanguage.json

```diff
@@ -43,19 +43,19 @@
-4|COMP-5|COM-REG|CONFIGURATION|CONTENT|CONTROLS|CONVENTION|CONVERTING|CRUNCH|DATE-COMPILED|DATE-WRITTEN|DAY-OF-WEEK|DB-KEY|DBNAME|DBNODE|DC|DEBUGGING|DEBUG-CONTENTS|DEBUG-ITEM|DEBUG-LINE|DEBUG-NAME|DEBUG-SUB-1|DEBUG-SUB-2|DEBUG-SUB-3|DECIMAL-POINT|DECLARATIVES|DEFAULT-DISPLAY|DELIMITED|DEPENDING|DEQUEUE|DESCENDING|DEST|DFHRESP|DFHVALUE|DICTNAME|DICTNODE|DISK|DISPLAY-1|DIVIDE|DIVISION|DOWN|DUPLICATE|DUPLICATES|EDIT|EGCS|EGI|EMI|ENCODING|END-ACCEPT|END-ADD|END-CALL|END-COMPUTE|END-DELETE|END-DIVIDE|END-EVALUATE|END-IF|END-MULTIPLY|END-OF-PAGE|END-PERFORM|END-READ|END-RECEIVE|END-RETURN|END-REWRITE|END-SEARCH|END-START|END-STRING|END-SUBTRACT|END-UNSTRING|END-WRITE|END-XML|ENDPAGE|ENQUEUE|EOP|ESI|EVALUATE|EVERY|EXHIBIT|EXITS|FD|FILE-CONTROL|FILLER|FINISH|FIND|FOOTING|FUNCTION-POINTER|GIVING|GOBACK|GREATER|GROUP-USAGE|HIGHLIGHT|HIGH-VALUE|HIGH-VALUES|ID|IDENTIFICATION|IDMS|IDMS-CONTROL|IDMS-RECORDS|IDMS-STATISTICS|IF|IGNORED|IMPLICIT|IN|INCREMENTED|INDEX|INDEXED|INDICATE|INITIAL|INITIALIZE|INITIATE|INPUT-OUTPUT|INSPECT|INSTALLATION|INTERNAL|INVOKED|I-O|I-O-CONTROL|JUST|JUSTIFIED|KANJI|KEPT|KEYBOARD|LESS|LEVELS|LIBRARY|LINAGE|LINAGE-COUNTER|LINES|LINE-COUNTER|LOADLIB|LOCAL-STORAGE|LONG-DATE|LONG-TIME|LONGTERM|LOWER|LOW-VALUE|LOW-VALUES|LTERM|LR|MAPS|MAP-BINDS|MANUAL|MEMBERS|MEMORY|MODIFIED|MODIFY|MODULES|MORE-LABELS|MULTIPLE|MULTIPLY|NATIONAL-EDITED|NATIVE|NEGATIVE|NETWORK|NODEADLOCK|NODENAME|NOLOCK|NORMAL|NOTIFICATION|NOTIFY|NOWRITE|NUMERIC-DATE|NUMERIC-EDITED|NUMERIC-TIME|OBJECT-COMPUTER|OBTAIN|OCCURS|ODT|OMITTED|ORDERLY|OTHER|OVERFLOW|OWN|PACKED-DECIMAL|PADDING|PAGE-COUNTER|PAGE-INFO|PARSE|PERFORM|PERMANENT|PIC|PICTURE|POINTER|POINTER-32|PORT|POSITIVE|PRINTER|PRINTING|PROCEDURES|PROCEDURE-POINTER|PROCEED|PROCESSING|PROGRAM-ID|PROTECTED|PTERM|PURGE|QUEUE|QUOTES|RANDOM|READER|READY|RECEIVED|RECORDING|RECURSIVE|REDEFINES|REEL|RELOAD|REMAINDER|REMOTE|REMOVAL|RENAMES|REPORT|REPORTING|REPORTS|RERUN|RESERVE|RETENTION|RETRIEVAL|RETURNING|RETURN-CODE|REVERSED|ROUNDED|RUN-UNIT|SAME|SAVE|SCRATCH|SCREENSIZE|SD|SEGMENT|SEGMENT-LIMIT|SELECTIVE|SENTENCE|SEQUENTIAL|SERVICE|SHIFT-IN|SHIFT-OUT|SHORT-DATE|SIGN|SORT|SORT-CONTROL|SORT-CORE-SIZE|SORT-FILE-SIZE|SORT-MERGE|SORT-MESSAGE|SORT-MODE-SIZE|SORT-RETURN|SOURCE-COMPUTER|SPACES|SPECIAL-NAMES|STORE|SQLIMS|SUBSCHEMA-AREANAMES|SUBSCHEMA-BINDS|SUBSCHEMA-CONTROL|SUBSCHEMA-CTRL|SUBSCHEMA-DESCRIPTION|SUBSCHEMA-DML-LR-DESCRIPTION|SUBSCHEMA-LR-CONTROL|SUBSCHEMA-LR-CTRL|SUBSCHEMA-LR-DESCRIPTION|SUBSCHEMA-LR-NAMES|SUBSCHEMA-LR-RECORDS|SUBSCHEMA-NAMES|SUBSCHEMA-RECNAMES|SUBSCHEMA-RECORDS|SUBSCHEMA-RECORD-BINDS|SUBSCHEMA-SETNAMES|SUBSCHEMA-NAME|STANDARD|STANDARD-1|STANDARD-2|STGID|STOP|STRING|SUBSCHEMA-NAMES|SUBTRACT|SUM|SYMBOLIC|SYNC|SYNCHRONIZED|SYSVERSION|TALLY|TALLYING|TAPE|TERMINATE|THAN|THREAD-LOCAL|THROUGH|THRU|TIMES|TODAYS-DATE|TODAYS-NAME|TOP|TRUNCATED|TYPEDEF|UNIT|UNSTRING|UP|UPGRADE|UPON|USAGE-MODE|UTF-8|VALIDATING|VIRTUAL|WHEN-COMPILED|WITH|WITHIN|WORDS|WORKING-STORAGE|WRITE-ONLY|ZERO|ZEROES|ZEROS|IS)(?![\\-\\w])",
```

Comment:
There are many verbs used for compiler options which are not here, e.g. NOCOMPILE, NOSSRANGE, LINECOUNT, ...

Reply:
Checking...

Reply:
Good catch

Reply:
This may be the [helpful link ](https://www.ibm.com/support/knowledgecenter/SS6SG3_6.2.0/pg/ref/rpcoshst.html)